x-pack/plugins/apm/public/hooks/use_search_strategy.ts

```diff
@@ -156,8 +156,8 @@ export function useSearchStrategy<
           setRawResponse(response.rawResponse);
           setFetchState({
             isRunning: response.isRunning || false,
-            loaded: response.loaded,
-            total: response.total,
+            ...(response.loaded ? { loaded: response.loaded } : {}),
+            ...(response.total ? { total: response.total } : {}),
```

Comment:
Wouldn't this suffice? 

```suggestion
            loaded: response.loaded ?? {},
            total: response.total ?? {},
```